pkg/types/types.go

```diff
@@ -76,3 +76,55 @@ type Status struct {
 	// newest node conditions in this field.
 	Conditions []Condition `json:"conditions"`
 }
+
```

Comment:
We should also add status unknown here. And set condition to unknown in condition manager.

Reply:
Will do.

Reply:
> And set condition to unknown in condition manager.

This requires another big change, imo. Will do this later in another PR.

Reply:
Sure. Please add TODO then.

Reply:
OK.